### PR TITLE
Update application management page to include filters and group actions

### PR DESCRIPTION
**Pull request by zwalchuk**
A number of changes to the application management page, including new screenshots, better info on deploying applications, filters, and group actions.

### CODE REVIEW

File: pages/management/applications.md [hunk 1]
```diff
@@ -4,76 +4,121 @@ title: Applications
...
+
+As the number of devices in your application grows, the device list will become increasingly busy. Filters provide a convenient way to quickly find specific devices based on shared characteristics. 
+
+### Add and update filters
+
+To add a filter, click the *Add filter* button near the top-left corner of your application page. You'll be presented with a window to configure your filter:
+
+<img src="/img/common/app/add-filter.png" width="80%">
+
+The first dropdown lists the device characteristics you can filter on. These characteristics correspond to the columns in your device list. The second dropdown contains comparison operators appropriate to the device characteristic, such as `is`, `is before`, `contains`, and `matches RegEx`. The third field is where you specify a value to filter on. Click *Add filter* to apply your configuration.
```
Comment (alisondavis17) [inline pages/management/applications.md]: this is true for now, but just a heads up that when we add tags, you will also be able to filter based on tags. I'm sure we will also create new docs for tags, and can update the filters docs as needed when the time comes.

File: pages/management/applications.md [hunk 2]
```diff
@@ -4,76 +4,121 @@ title: Applications
...
+### Public URL
 
-### Reboot All Devices
+This option enables a public URL for all devices in the application. For more details on public URLs, please refer to the [device page][device-page].
 
-This action allows you to perform a reboot of all the devices in the fleet/application. This is different from the `Restart Application` action mentioned above, because in this action, the entire device including the kernel will be rebooted as if there was a power cycle. It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of the action it missed.
+### Restart Application
 
-__Warning:__ This action is only supported on devices with an Agent version >= 1.1.0
+The *Restart Application* action will restart the application container on selected devices that are currently online. It should be noted that these action notifications are not queued up, so if a device is offline when the action is triggered the application not be restarted when it comes back online.
```
Comment (alisondavis17) [inline pages/management/applications.md]: missing "will" before "not be restarted"

File: pages/management/applications.md [hunk 3]
```diff
@@ -4,76 +4,121 @@ title: Applications
...
 
-The `Restart Application` action is a fleet wide action that will restart the application container on all the devices that are currently online. It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of it.
+## Application actions
 
-## Dangerous Application Actions
+Actions let you change the state of some or all of the devices in your application. They can be applied in two ways:
 
-### Purge Data (on all devices)
+1. The *Actions* menu, located on the left side of the application page, allows you to apply state changes to all devices in the application.
+2. *Group Actions*, found at the top-right of the device list, are applied to a subset of the devices in your fleet. You can specify which devices will be affected by clicking the check boxes on the left of the device list.
```
Comment (alisondavis17) [inline pages/management/applications.md]: Should we be more explicit that you can also use filters/views to apply group actions to a subset of devices?

File: pages/management/applications.md [hunk 4]
```diff
@@ -4,76 +4,121 @@ title: Applications
...
+You will be asked to choose a name for the saved view. This view can then be selected at any time by clicking the *Views* button above the filter box.
 
-The `Restart Application` action is a fleet wide action that will restart the application container on all the devices that are currently online. It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of it.
+## Application actions
 
-## Dangerous Application Actions
+Actions let you change the state of some or all of the devices in your application. They can be applied in two ways:
 
-### Purge Data (on all devices)
+1. The *Actions* menu, located on the left side of the application page, allows you to apply state changes to all devices in the application.
```
Comment (alisondavis17) [inline pages/management/applications.md]: A product question for @shaunmulligan - is it superfluous (and maybe even confusing) to have this on the application page, when we also have Group Actions to the top right of the device list, which could easily be applied to your entire fleet? Should we combine the two somehow? Or remove any confusing overlap? Right now it's not clear where you should go first to apply an action to your entire fleet. 

The actions that appear in both places are: restart app, reboot, purge, shutdown. 
The actions that are unique to the actions page are: public url, support access, remove collaborators

File: pages/management/applications.md (hunk 3 above)
Comment (zwalchuk) [reply pages/management/applications.md]: Yes, that makes sense. I'll add a sentence along those lines.

**Pull request merged**